had troubles with mult2.py
